site/src/main/java/uk/nhs/digital/common/components/SearchComponent.java

```diff
@@ -215,8 +217,7 @@ private Constraint publicationSystemConstraint(String query) {
         return or(
             constraint("publicationsystem:Title").contains(query),
             constraint("publicationsystem:Summary").contains(query),
-            constraint("publicationsystem:KeyFacts").contains(query),
-            constraint("common:SearchableTags").contains(query)
+            constraint("publicationsystem:KeyFacts").contains(query)
```

Comment:
looks like a merge gremlin, please revert and include `common:SearchableTags`

Reply:
No it's pulled in from the 'commonConstraint' function - see earlier conversations about this.